apiserver/src/api/mod.rs

```diff
@@ -130,6 +131,18 @@ pub async fn run_server<T: 'static + BottlerocketShadowClient>(
     );
 
     // Set up the actix server.
+    let server_addr;
```

Comment:
Nit:

```suggestion
    let server_addr;
    
    let server_addr = if k8s_service_addr {
    } else {
    }
```

I wonder if you should  validate the IP format?

Reply:
Which IP format? If it is for k8s service, we don't need validation since we just want to get the ip family information for the cluster.